api/proto/controller/v1/controller_service.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package controller.v1;
```

Comment:
We probably shouldn't use a v1/v2 postfix for the controller.

Reply:
Even if we eventually have new "versions" of the controller API, since this is internal we can make breaking changes as long as we update the entire disperser/full node at the same time.